packages/blocks/src/page-block/default/default-page-block.ts

```diff
@@ -234,12 +234,8 @@ export class DefaultPageBlockComponent extends LitElement implements BlockHost {
   private _removeHotkeys() {
     removeCommonHotKey();
     hotkey.removeListener([
-      HOTKEYS.UNDO,
```

Comment:
It seems that some listeners have been removed in `removeCommonHotKey`, is it still needed here? 🤔

Reply:
Looks good to avoid duplication here!